### DISCUSSION TITLE
Sensitivity Analysis

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#904] [Questions & Problems] [answered]
For sensitivity analysis of PK parameters, what exactly do the output values represent for each parameter (sensitivity coefficients?)

**Comment by msevestre** [comment, marked as answer]
hi @pburkat 
Check the documentation out 
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/sensitivity-analysis

The section on Sensitivity Analysis is nicely written I believe
